x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4

```diff
@@ -151,11 +151,21 @@ aggField
     ;
 
 qualifiedName
+    : {this.isDevVersion()}? OPENING_BRACKET qualifier=UNQUOTED_IDENTIFIER? CLOSING_BRACKET DOT OPENING_BRACKET name=fieldName CLOSING_BRACKET
```

Comment:
This is a quiet verbose syntax that requires wrapping an identifier with brackets rather than just pre-pending it with qualifier when changing existing query.

I wonder about the following alternatives:
* `[qualifier].identifier`
* `qualifier -> identifier`

They are a bit shorter. Could they be ambigous?